src/Serilog.Sinks.MSSqlServer/Sinks/MSSqlServer/SqlTableCreator.cs

```diff
@@ -5,150 +5,152 @@
 
 namespace Serilog.Sinks.MSSqlServer
 {
-	internal class SqlTableCreator
```

Comment:
The diff in this file appears to be primarily whitespace, would it be possible to undo the whitespace changes so the diff is clearer?